bee-network/bee-autopeering/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "bee-autopeering"
-version = "0.3.0"
+version = "0.4.0-alpha1"
```

Comment:
Let's not put alpha if we're using it right now for bee-0.3

Reply:
so, you mean we should release the crate with ipv6 as `0.4.0`, and then put the next changes/fixes/features in a `0.5.0-alphaX`?